modules/popup-stack/lib/PopupStack.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * This type is purposely an interface so that it can be extended for a
+ * specific use-case
+ */
+export interface PopupStackItem {
+  element: HTMLElement;
+}
+
+function getLast<T>(items: T[]): T | null {
+  if (items.length) {
+    return items[items.length - 1];
+  }
+  return null;
+}
+
+/**
+ * Set the z-index of each item in the stack by changing setting
+ * `element.style.zIndex`. This is a framework-agnostic approach to changing
+ * zIndex. This strategy will start the popup stack at 30 and each popup will
+ */
+export function defaultGetValue(index: number, length: number): number {
```

Comment:
Couldn't overriding this function make the behavior inconsistent? Can you remind me of the use case for this?

Reply:
If we've removed the ability to change this, can we rename it to `getValue` or bake its behavior into `setZIndexOfElements`?

Reply:
I have tests for this function, so I'm not sure it makes sense to bake it into `setZIndexOfElements`. I could rename it to `getValue`.